api/routes/onboarding.py

```diff
@@ -10,20 +16,85 @@ def onboarding():
     """
     docstring
     """
-    return "Onboarding API. This API endpoint is used for onboarding users and stokvels ( on creation )"
-
+    return render_template("onboarding_template.html")
 
 @onboarding_bp.route(f"{BASE_ROUTE}/users", methods=["POST"])
 def onboard_user():
     """
-    docstring
+    Handles onboarding of a new user.
     """
-    return "Complete Onboarding endpoint"
+    try:
+        name = request.form["name"]
+        surname = request.form["surname"]
+        cell_number = request.form["cellphone_number"]
+        id_number = request.form["id_number"]
+
+        flag = queries.check_if_number_exists_sqlite(cell_number)
```

Comment:
This can directly be put into an if, for example

if check_if_number_exists():

Reply:
Also we would not need to check this at this stage, the check if number exists is handled in the whatsapp controller

Reply:
Should we keep this check in for now while the WhatsApp is not 100% up and running yet?